spatial/r3/mat_safe.go

```diff
@@ -60,6 +60,8 @@ func Eye() *Mat {
 //                  ⎡ 0 -z  y⎤
 //  Skew({x,y,z}) = ⎢ z  0 -x⎥
 //                  ⎣-y  x  0⎦
+//
+// DEPRECATED: use Mat.Skew()
```

Comment:
@kortschak It's a matter of convenience but couldn't we have both?

Reply:
We've deprecated things from r2 and r3 in the past (though looking at the package I see that they are still there). There is an weird asymmetry in the deprecation (justified by implementation) in that the previous deprecation was a change from method to package function and this one is from package function to method (the implementation detail is that those are non-heap allocating returns, while this one is a heap allocating return). @sbinet?